docs/guides/circuits.rst

```diff
@@ -163,13 +169,16 @@ Appending individual gates to circuits is also possible, as we'll see when we bu
   q_2: ┤ Rx($\gamma$) ├
        └──────────────┘
 
-To build up the mixing circuit, we ust need to put parametrized RX gate on each of the qubits. We can do that with a for loop:
+To build up the mixing circuit, we need to put parametrized RX gate on each of the qubits. We can do that with a for loop:
 
 .. literalinclude:: /examples/circuits_guide.py
   :language: python
   :start-at: mixing_circ = Circuit()
   :end-at: ic(mixing_circ)
 
+.. note::
+  ``orquestra-quantum`` comes with a `built-in utility function <https://github.com/zapatacomputing/orquestra-quantum/blob/main/src/orquestra/quantum/circuits/_generators.py#L14=>`_ that puts down a layer of the same single-qubit gate acting on all qubits.
```

Comment:
I'd say it's better to put a link to the docs rather than to the source code?
Also, I'm not sure if GitHub permalinks are good thing to use in docs.
But I truly don't know, feel free to ignore :)